plugins/woocommerce/tests/api-core-tests/tests/system-status/system-status-crud.test.js

```diff
@@ -374,14 +404,36 @@ test.describe( 'System Status API tests', () => {
 				] ),
 			} )
 		);
-		expect( responseJSON ).toEqual(
-			expect.objectContaining( {
-				dropins_mu_plugins: expect.objectContaining( {
-					dropins: [],
-					mu_plugins: [],
-				} ),
-			} )
-		);
+
+		// local environment differs from external hosts.  Local listed first.
+		if ( ! shouldSkip ) {
+			expect( responseJSON ).toEqual(
+				expect.objectContaining( {
+					dropins_mu_plugins: expect.objectContaining( {
+						dropins: [],
+						mu_plugins: [],
+					} ),
+				} )
+			);
+		} else {
+			expect( responseJSON ).toEqual(
```

Comment:
External environment has a couple of drop-in plugins.  So just need to deal with those.